k8sclient/k8sclient.go

```diff
@@ -339,12 +345,26 @@ func getKubernetesDelegate(client KubeClient, net *types.NetworkSelectionElement
 		return nil, logging.Errorf("getKubernetesDelegate: failed to get the netplugin data: %v", err)
 	}
 
+	// Get resourceName annotation from NetDefinition
+	deviceID := ""
+	resourceName, ok := customResource.Metadata.Annotations[resourceNameAnnot]
```

Comment:
Shall we handle the case when multiple resourceNames be added in the pod spec?

for example, given the annotations below:

k8s.v1.cni.cncf.io/resourceName: "intel.com/sriov", "kernel.org/virt"; 
or
k8s.v1.cni.cncf.io/resourceName: "intel.com/sriov"
k8s.v1.cni.cncf.io/resourceName: "kernel.org/virt"

the annotations for resourceName will be a list of resource types, then there might need a loop against resourceName for each type of resources?